### ISSUE TITLE
question about code in "googlenet.py"

### ISSUE THREAD

**Issue by kdwonn** [Confusezius/Deep-Metric-Learning-Baselines#19]
Thanks for your code and recent researches. 

The question is, it seems like code presented in [https://github.com/Confusezius/Deep-Metric-Learning-Baselines/blob/master/googlenet.py](url) is for Inception-BN, not googlenet. Am I missing something?

**Comment by kdwonn** [comment]
Got an answer from pytrorch issues. - [https://github.com/pytorch/vision/issues/906](url)
So "googlenet.py" is not the same as the original "googlenet" presented in "Going Deeper .. ". It has different kernel sizes and even uses BN.

I think I should reference Tensorflow/Pytorch googlenet code instead of "Going deeper .. " in my next paper.

**Comment by Confusezius** [comment]
Thanks for noticing! I just took the GoogLeNet from the official Torchvision Repo, since they also reference the original paper, but you are right. I will add a disclaimer for that :).
